polling_stations/templates/base_full.html

```diff
@@ -23,7 +23,7 @@
   <meta name="twitter:site" content="@democlub">
   <meta property="og:image" content="{{ CANONICAL_URL }}{% block og_image %}{% static "dc_theme/images/logo.png" %}{% endblock og_image %}">
   <meta property="og:title" content="{% block og_title %}{{ site_title }}{% endblock og_title %}">
-  <meta property="og:description" content="{% block og_description %}Find out where to vote{% endblock og_description %}">
+  <meta property="og:description" content="{% block og_description %}Find out where to vote on 6 May 2021{% endblock og_description %}">
```

Comment:
Can this be based off `NEXT_CHARISMATIC_ELECTION_DATE`?

e.g:

1. add to [`global_settings`](https://github.com/DemocracyClub/UK-Polling-Stations/blob/master/polling_stations/apps/pollingstations/context_processors.py#L8)
2. `{% if NEXT_CHARISMATIC_ELECTION_DATE %} on {{ NEXT_CHARISMATIC_ELECTION_DATE|date }}{% endif %}`

Reply:
Had a bash at an implementation in 005765f. As discussed on slack `NEXT_CHARISMATIC_ELECTION_DATE` is a string, so had to be cast to a date in the context processor.